### PR TITLE
Using getFacilityByEntityId in all possible places

### PR DESCRIPTION
**Pull request by BaranekD**

### CODE REVIEW

**Comment by BaranekD** [comment]
I've done some changes, ready to merge (or at least review :))

File: lib/Auth/Process/ForceAup.php [hunk 1]
```diff
@@ -118,7 +118,7 @@ public function process(&$request)
         }
 
         try {
-            $facilities = $this->adapter->getFacilitiesByEntityId($request['SPMetadata']['entityid']);
+            $facilities = $this->adapter->getFacilityByEntityId($request['SPMetadata']['entityid']);
```
Comment (pajavyskocil) [inline lib/Auth/Process/ForceAup.php]: Method getFacilityByEntityId returns object Facility, not array!

File: lib/AdapterRpc.php [hunk 2]
```diff
@@ -304,37 +300,36 @@ public function getFacilityAttribute($facility, $attrName)
...
     {
-        $facilities = $this->connector->get('facilitiesManager', 'getFacilitiesByAttribute', [
-            'attributeName' => 'urn:perun:facility:attribute-def:def:entityID',
-            'attributeValue' => $spEntityId,
-        ]);
+        $facility = $this->getFacilityByEntityId($spEntityId);
 
-        $groups = [];
+        if ($facility === null) {
+            return [];
```
Comment (pajavyskocil) [inline lib/AdapterRpc.php]: If you move line 309 upper, you can return $groups here.

File: lib/AdapterLdap.php [hunk 3]
```diff
@@ -226,6 +231,31 @@ public function getFacilitiesByEntityId($spEntityId)
         // TODO: Implement getEntityByEntityId() method.
     }
 
+    public function getFacilityByEntityId($spEntityId)
+    {
+        $facilities = $this->connector->searchForEntity(
```
Comment (pajavyskocil) [inline lib/AdapterLdap.php]: Using another name than $facilities (for example $facilityArray or $ldapResult) will be better.

**Comment by BaranekD** [comment]
Requested changes done

**Pull request merged**